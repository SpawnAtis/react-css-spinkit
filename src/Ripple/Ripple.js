import React from 'react'
import styles from './styles.css'

const Ripple = () => (
  <>
    <style>{styles.toString()}</style>
    <div className="lds-ripple">
      <div />
      <div />
    </div>
  </>
)

export default Ripple
