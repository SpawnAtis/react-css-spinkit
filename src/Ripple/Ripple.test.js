import React from 'react'
import {render} from 'react-testing-library'
import Ripple from './Ripple'

describe('Ripple component', () => {
  it('Should render Ripple without problems', () => {
    const {asFragment} = render(<Ripple />)
    expect(asFragment()).toMatchSnapshot()
  })
})
