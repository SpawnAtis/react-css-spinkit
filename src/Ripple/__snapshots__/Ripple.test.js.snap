// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ripple component Should render Ripple without problems 1`] = `
<DocumentFragment>
  <style>
    [object Object]
  </style>
  <div
    class="lds-ripple"
  >
    <div />
    <div />
  </div>
</DocumentFragment>
`;
